### PR TITLE
Fix heredoc within parens formatting causing subsequent syntax error

### PR DESCRIPTION
**Pull request by mjago**
Fix for issue #88

### CODE REVIEW

**Comment by mjago** [comment]
@gingermusketeer  @splattael  @danielma Any input on this PR guys? This is a fix for a reported bug that leaves code broken so I think it should be considered.

I will probably focus on fixing bugs on the new formatter moving forward - but I think it was important to fix these breaking bugs on current formatter. Style issues on current formatter are less urgent than getting new formatter production (beta) ready. Also, Ripper changes need to be followed up to ensure we know what is coming around the corner :smile:

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 ### Fixed
+- Fix `quote_style` Bug: Rufo breaks HEREDOC syntax. (issue [#88](https://github.com/ruby-formatter/rufo/issues/88)). This fixes heredoc within parens formatting causing bad formatting and subsequent syntax error.
```
Comment (gingermusketeer) [inline CHANGELOG.md]: What does this change have to do with `quote_style`? Was the bug introduced by the change that added it?

**Comment by gingermusketeer** [comment]
@mjago Getting this sorted is great. Nice for folks to be able to use this soon! Perhaps we should do a release after this is merged?

File: CHANGELOG.md (hunk 1 above)
Comment (mjago) [reply CHANGELOG.md]: Oops ~ I thought `quote_style` was some fancy markdown formatting that was being used in the changelog (cut-paste error) :blush: I'll fix that

**Comment by mjago** [comment]
@splattael do you wanna do a release?

**Comment by gingermusketeer** [comment]
@mjago All your changes have been released with v0.4.0 :)

**Pull request merged**